chapter_convolutional-neural-networks/pooling.md

```diff
@@ -303,7 +303,7 @@ pool2d(X)
 1. What is the computational cost of the pooling layer? Assume that the input to the pooling layer is of size $c\times h\times w$, the pooling window has a shape of $p_h\times p_w$ with a padding of $(p_h, p_w)$ and a stride of $(s_h, s_w)$.
 1. Why do you expect maximum pooling and average pooling to work differently?
 1. Do we need a separate minimum pooling layer? Can you replace it with another operation?
-1. Is there another operation between average and maximum pooling that you could consider (hint: recall the softmax)? Why might it not be so popular?
+1. Is there another operation besides average and maximum pooling that you could consider (hint: recall the softmax)? Why it may not be popular?
```

Comment:
"between" means something (e.g., softmax) in a spectrum where average pooling and max pooling are two extremes.

"Why it may not be popular" is grammatically wrong.

Reply:
In Google Search, "Why may it not be" returns 1 million results, while "Why it may not be" returns 84 millions...

Reply:
Well, we cannot simply use the number of results to judge the correctness.
"Why may it not be popular?" is for asking questions.
"Here's why it may not be popular." is for noun phrases, usually part of a sentence.

What if part-of-sentences are naturally more frequent than questions in natural language?

Reply:
Make sense to me now! How about "What could be the reason for it not to be popular?" Since double negative is rarely used in standard English...